workspaces/orchestrator/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@internal/orchestrator",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "dev": "yarn workspaces foreach -A --include backend --include app --parallel -v -i run start",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
+    "build:all": "backstage-cli repo build --all",
+    "build:api-reports": "yarn build:api-reports:only",
+    "build:api-reports:only": "backstage-repo-tools api-reports --allow-all-warnings -o ae-wrong-input-file-type --validate-release-tags",
+    "clean": "backstage-cli repo clean",
+    "test": "backstage-cli repo test",
+    "test:all": "backstage-cli repo test --coverage",
+    "fix": "backstage-cli repo fix",
+    "lint": "backstage-cli repo lint --since origin/main",
+    "lint:all": "backstage-cli repo lint",
+    "prettier:check": "prettier --check .",
+    "new": "backstage-cli new --scope @red-hat-developer-hub",
+    "postinstall": "cd ../../ && yarn install"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "plugins/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redhat-developer/rhdh-plugins",
+    "directory": "workspaces/orchestrator"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.29.0",
+    "@backstage/e2e-test-utils": "^0.1.1",
+    "@backstage/repo-tools": "^0.10.0",
+    "@changesets/cli": "^2.27.1",
+    "@spotify/prettier-config": "^12.0.0",
+    "node-gyp": "^9.0.0",
+    "prettier": "^2.3.2",
+    "typescript": "~5.3.0"
+  },
+  "dependencies": {
+    "@ianvs/prettier-plugin-sort-imports": "^4.4.0",
+    "knip": "^5.27.4"
```

Comment:
Could you add these as devDependencies instead of dependencies?

Reply:
These were added by the migration script, I'm actually not sure what they are for...
They are in dependencies for the other plugins
@BethGriggs should these be in devDependencies?

Reply:
I have a PR up for shifting them to devDependencies for other plugins as well: https://github.com/redhat-developer/rhdh-plugins/pull/55 
knip is for adding reports on unused dependencies: https://knip.dev/
@ianvs/prettier-plugin-sort-imports I think was included to run prettier

Reply:
Thanks! Fixed